logic/src/commonJvmAndroid/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```
@@ -99,7 +99,7 @@ import java.util.Collections
 
 @Suppress("LongParameterList", "TooManyFunctions")
 class CallManagerImpl internal constructor(
-    private val calling: Calling,
+    private val calling: Calling?,
```

Now the whole calling may not work in production and it will fail silently?